docs/getting-started-guides/ubuntu/index.md

```diff
@@ -27,6 +27,13 @@ sudo snap install conjure-up --classic
 conjure-up kubernetes
 ```
 
+As well as Homebrew for macOS:
+
+```
+brew install conjure-up
```

Comment:
It might be worthwhile to note that you need to refresh brew `brew update` -- that happened to me this morning but it's also fairly well known to the brew community. Non-blocking comment, just food for thought.